src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseFlags.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.PowerFx.Core.Texl.Intellisense
+{
+    /// <summary>
+    /// Flags used to control Intellisense bahavior.
+    /// </summary>
+    [Flags]
+    public enum IntellisenseFlags
+    {
+        /// <summary>
+        /// Default bahavior
+        /// </summary>
+        Default = 0x0,
+
+        /// <summary>
+        /// Suggests unqualified enums when possible
+        /// </summary>
+        SuggestUnqualifiedEnums = 0x1,
+    }
```

Comment:
Can this by tied to the resolver?
Ie, somehow in PowerFx standalone, we already make a decision to require qualified enums ("TimeUnit.Hours"). 
(Whereas canvas apps still allows unqualified enums ("Hours"). 

Can we leverage that mechanism and avoid adding flags?